README.md

```diff
@@ -1,45 +1,122 @@
 
 ```
-$ yarn build
+$ git checkout master
+$ git pull upstream master && git push origin master
+```
+
+Then, create a branch to work on.
+
+```
+$ git checkout -b your-branch
+```
+
+Now you can make your changes. See the next section about building and viewing the site locally for those steps.
+
+### Step 4: Submit a pull request
+
+Once you are ready to submit your changes, create a pull request.
+
+```
+git push -u origin your-branch
+```
+
+This will create the branch on your GitHub project fork.
+
+**Note** that the -u flag only needs to be used the first time you create a pull request for this branch.
+
+Next, visit the project fork in your GitHub account and locate the new branch, and click "Compare and pull request".
+
+Ensure the "base fork" points to the correct repository and branch and provide a clear title and explanation of the changes with links to any issues that are being fixed by the change.
+
+Then click "Create pull request".
+
+## Building and viewing the site locally
+
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/).
+
+Check out their [installation requirements](https://v2.docusaurus.io/docs/installation#requirements) for tooling prerequisites.
+
+### Step 1: Initialize yarn
+
+The first step is to initialize the use of Yarn to build the site locally.
+
+From your terminal, change directory to the project root and run:
+
+```
+$ yarn
 ```
 
-This command generates static content into the `build` directory and can be served using any static contents hosting service.
+### Step 2: Build the website
 
-### Versioning
+The following command generates static content into the `build` directory which can be served using any static contents hosting service.
 
 ```
-$ yarn run docusaurus docs:version 1.1.0
+$ yarn build
 ```
 
-This command creates a copy of /docs and tags with the specified version
+It also ensures that the site is building correctly without errors or misconfigurations.
+
+### Step 3: View the site in localhost
 
-### Deployment
+You can view the site via localhost in your browser by running this command from inside the project:
 
 ```
-$ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
+$ yarn start
 ```
 
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+This command starts a local development server and opens up a browser window.
+
+Most changes to the site are reflected instantly without having to restart the server.
+
+You can run `yarn start` independently of `yarn build`.
+
+## When will my changes be reflected in docs.temporal.io?
+
+As soon as your pull request is merged, a new build will automatically kick off and deploy the changes to the [next](https://docs.temporal.io/docs/next/) version of the docs.
+
+Temporal will accumulate changes in the next version and periodically release them to the "current" version of the docs.
```

Comment:
Can you reword this to indicate that our team manually does this and it's not some automatic CRON (periodically is ambiguous basically)